crates/fuel-core-nats/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "fuel-core-nats"
```

Comment:
can you change for `fuel-data-stream` as we talked in the last sync?

Reply:
ok, let's go with this one as it is, then after the tech spec and more clarity about crates and etc we can bike-shedding about the name again